Frends.Community.SQL.Tests/SQLTaskTest.cs

```diff
@@ -39,7 +39,7 @@ public void DataReaderToCsvTest_AllColumns()
                 var entries = SQL.DataReaderToCsv(reader, csvFile, options, new System.Threading.CancellationToken());
                 csvFile.Flush();
                 var result = writer.ToString();
-                var resultLines = result.Split(new String[] { "\r\n" }, StringSplitOptions.None);
+                var resultLines = result.Split(new string[] { "\r\n" }, StringSplitOptions.None);
 
                 // 4 lines = 1 header line + 2 data lines + 1 newline at end of file
                 Assert.AreEqual(4, resultLines.Length);
```

Comment:
https://github.com/CommunityHiQ/Frends.Community.SQL/blob/a6cf9c0a09d1b7106af3915e89ea8e5e18f6b40f/Frends.Community.SQL.Tests/SQLTaskTest.cs#L179
No need to write to console.